docs/change-log/index.md

```diff
@@ -717,38 +717,57 @@ with a drill-down to view the steps and their details. [Tech Preview]
-|ML-3640| When running a remote function/workflow, the `context` global parameter is not automatically injected.| Use `get_or_create_ctx`| v1.3.0    |
-|ML-4153| When creating a passthrough features-set in the UI, with no online target, the feature-set yaml includes a parquet offline target, which is ignored.| NA | v1.4.0  |
-|ML-4442| After a model is deployed without applying monitoring (`set_tracking()` was not set on the serving function), monitoring cannot be added.|Delete the existing model endpoint (`mlrun.get_run_db().delete_model_endpoint()`), then redeploy the model.| v1.5.0 |
-|ML-4740| When running function `batch_inference_v2` from the SDK, the `ingest()` function accepts 3 parameters as Data-item or other types: `dataset`, `model_path` and `model_endpoint_sample_set`. If you provided these parameters as non Data-items and later on you want to rerun this function from the UI, you need to provide these parameters as Data-item.|Prepare suitable Data-item and provide it to the batch-rerun UI.| v1.5.0    |
-|ML-4758| In rare cases, deleting a heavy project is unsuccessful and results in a timeout error message while the project moves to offline state.| Delete again.| v1.5.0    |
-|ML-4907|MLRun Client does not support Win OS.|NA|v1.3.0| 
+|ML-3315|Spark ingestion does not support nested aggregations.                 |NA |v1.2.1    |
+|ML-3386|Documentation is missing full details on the feature store sources and targets.| NA|v1.2.1    |
+|ML-3445|`project.deploy_function` operation might get stuck when running v1.3.0 demos on an Iguazio platform running v3.2.x.| Replace code: `serving_fn = mlrun.new_function("serving", image="python:3.9", kind="serving", requirements=["mlrun[complete]", "scikit-learn~=1.2.0"])` with: <br>`function = mlrun.new_function("serving", image="python:3.9", kind="serving") function.with_commands([ "python -m pip install --upgrade pip", "pip install 'mlrun[complete]' scikit-learn==1.1.2", ])`|v1.3.0    |
+|ML-3480|Documentation: request details on label parameter of feature set definition.| NA | v1.2.1    |
+|NA|The feature store does not support schema evolution and does not have schema enforcement.| NA| v1.2.1    |
+|ML-3633|Fail to import a context from dict                                     |When loading a context from dict (e.g.: mlrun.MLClientCtx.from_dict(context)), make sure to provide datetime objects and not string. Do this by executing `context['status']['start_time'] = parser.parse(context['status']['start_time'])<br> context['status']['last_update'] = parser.parse(context['status']['last_update'])` prior to `mlrun.MLClientCtx.from_dict(context)`|v1.3.0    |
```

Comment:
The description here is too generic, need better explanation

Reply:
moved to fixed in v1.4.0